### PR TITLE
Added a rule about avoiding the semi-colon for statement terminationg

### PR DESCRIPTION
**Pull request by Haud**
@ThibaultKlein @paulmiard

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,23 @@ If there is anything you are looking for that is not covered here you should ref
...
+let myVar = 0
+doSomething(myVar)
+
+return
+```
+
+Avoid multiple statements on a single line.
+
+```swift
+guard let obj = myObj else { print("Something went wrong"); return; } // Wrong! Instead, place each item on its own line.
```
Comment (ThibaultKlein) [inline README.md:27]: are we sure about that? Sometimes I like putting everything on the same line if it's still easy to read, especially when you don't need to have a print

`guard let a = a else { return }`
Thoughts?

Comment (paulmiard) [inline README.md:27]: then it would be an exception and it would confuse the rule itself, as well as some auto-formatting tool

**Comment by paulmiard** [comment]
:+1:

**Comment by ThibaultKlein** [comment]
:+1:

File: README.md (hunk 1 above)
Comment (ThibaultKlein) [inline README.md:27]: Sounds good

**Pull request merged**